packages/block-library/src/index.js

```diff
@@ -124,10 +123,10 @@ export const registerCoreBlocks = () => {
 			return;
 		}
 		const { metadata, settings, name } = block;
-		if ( metadata ) {
-			unstable__bootstrapServerSideBlockDefinitions( { [ name ]: metadata } ); // eslint-disable-line camelcase
-		}
-		registerBlockType( name, settings );
+		registerBlockType( name, {
+			...metadata,
+			...settings,
+		} );
```

Comment:
Any particular reasons for avoiding `unstable__bootstrapServerSideBlockDefinitions`? I guess the original idea was that this is the API that will be used once we move these to the server.

Reply:
Right, I did it as an experiment and there are no differences at all. I will revert to `unstable__bootstrapServerSideBlockDefinitions` until we have server-side covered 👍

Reply:
Reverted 👍